Packs/SumoLogic_Cloud_SIEM/Integrations/SumoLogicCloudSIEM/SumoLogicCloudSIEM.py

```diff
@@ -731,6 +769,7 @@ def main() -> None:
     assert isinstance(first_fetch_timestamp, int)
 
     fetch_query = demisto.params().get('fetch_query')
+    record_summary_fields = demisto.params().get('record_summary_fields')
```

Comment:
```suggestion
    record_summary_fields = demisto.params().get('record_summary_fields') or RECORD_SUMMARY_FIELDS_DEFAULT
```

Reply:
Done. Removed `RECORD_SUMMARY_FIELDS_DEFAULT` in favor of `defaultvalue` as suggested